packages/accordions/src/elements/accordion/components/Header.tsx

```diff
@@ -11,6 +11,9 @@ import ChevronDown from '@zendeskgarden/svg-icons/src/16/chevron-down-stroke.svg
 import { useAccordionContext, useSectionContext, HeaderContext } from '../../../utils';
 import { StyledHeader, StyledRotateIcon, COMPONENT_ID as buttonGardenId } from '../../../styled';
 
+/**
+ * @extends HTMLDivElement
```

Comment:
Should be `HTMLAttributes<HTMLDivElement>`

Reply:
Fixed.

Reply:
extends have been removed for now.